modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AccountAddressResourceImpl.java

```diff
@@ -242,7 +236,7 @@ public AccountAddress postAccountByExternalReferenceCodeAccountAddress(
 
 		AccountEntry accountEntry =
 			_accountEntryService.fetchAccountEntryByExternalReferenceCode(
```

Comment:
sorry @matyaswollner, probably I was not clear... here is fine to have the get (instead of the fetch) but before you moved the get below in the method. I would like to have the get here